code/modules/clothing/masks/gasmask.dm

```diff
@@ -37,6 +37,7 @@
 		. += "<span class='notice'>[src] has [max_filters] slot\s for filters.</span>"
 	if(LAZYLEN(gas_filters) > 0)
 		. += "<span class='notice'>Currently there [LAZYLEN(gas_filters) == 1 ? "is" : "are"] [LAZYLEN(gas_filters)] filter\s with [get_filter_durability()]% durability.</span>"
+		. += "<span class='notice'>The filters can be removed by right-clicking with an empty hand on [src].</span>"
```

Comment:
The text says right clicking with an empty hand. Isn't `attack_self_secondary` from being in someone's active hand, and right clicked? Rather then right clicking with an empty hand, which is maybe a bit more intuitive

Reply:
Well it works if you click it with an empty hand too, but if you want I'll change it